pkg/job/job.go

```diff
@@ -15,9 +16,11 @@ import (
 var jobTargetRexexp = regexp.MustCompile(`release-openshift-(ocp|origin)-installer-(.*)-\d.\d`)
 
 type Job struct {
-	FullName string
-	Target   string
-	ID       string
+	FullName       string
+	ID             string
+	ComputedResult string
+	UserName       string
```

Comment:
Having the username here looks like an interface segregation violation to me.

Can we have the `ToHTML` method accept the username as an argument instead?